src/vue/backend/src/views/GuidedSetup.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+    <idx-fullscreen customClass="guided-intro">
+        <idx-container fluid>
+            <idx-block className="hero">
+                <h1>Welcome to IMPress for IDX Broker</h1>
+                <p>Add some description about how we will guide the user through setting up IMPress</p>
+            </idx-block>
+            <idx-block className="media">
+                <idx-block className="media-image">
+                    <img src="@/assets/guided-setup.svg" alt="Illustration of monitor and mobile device" loading="lazy">
+                </idx-block>
+                <idx-block className="media-content">
+                    <h2>What this guide covers</h2>
+                    <idx-block>Cover the bases paragraph on what the install covers.</idx-block>
+                    <idx-block>Lorem ipsum dolor sit amet,</idx-block>
+                    <idx-list>
+                        <idx-list-item>Bullet list item 1</idx-list-item>
+                        <idx-list-item>Bullet list item 2</idx-list-item>
+                        <idx-list-item>Bullet list item 3</idx-list-item>
+                    </idx-list>
+                    <idx-button tag="a" href="#" size="lg">Let's Get Started</idx-button>
+                </idx-block>
+            </idx-block>
+        </idx-container>
+    </idx-fullscreen>
+</template>
+
+<style scoped lang="scss">
+    @import '~@idxbrokerllc/idxstrap/dist/styles/globalVariables.scss';
+    @import '~@idxbrokerllc/idxstrap/dist/styles/base.scss';
```

Comment:
These are already imported, global variables is imported in `vue.config.js` and the base styles are imported in `main.js`